### DISCUSSION TITLE
Auto-Type in VMs

### DISCUSSION THREAD

**Discussion by thewalkerz** [keepassxreboot/keepassxc#8361] [Q&A] [answered]
Hi, hope you're feel so good today.

 Is it possible to do an auto-type with virtual machines? I found this placeholder in the keepassxc's wiki, but I don't know how to use it:
`{MODE=VIRTUAL}`

I'm using Hyper-V.

Thanks for this amazing password manager :)

**Comment by droidmonkey** [comment, marked as answer]
Put that placeholder at the beginning of your sequence. You can also use the drop down menu in the Auto-Type selection dialog and choose "Use Virtual Keyboard" on Windows. 

https://keepassxc.org/docs/KeePassXC_UserGuide.html#_performing_global_auto_type
